source/postprocess/dynamic_topography.cc

```diff
@@ -110,9 +115,14 @@ namespace aspect
               // for each of the quadrature points, evaluate the
               // stress and compute the component in direction of the
               // gravity vector
+              
+              double dynamic_topography_x_volume = 0;
+              double volume = 0;
+              Point<dim> location = fe_values.quadrature_point(0);
+
               for (unsigned int q=0; q<quadrature_formula.size(); ++q)
```

Comment:
Yes, we could, but I actually intentionally kept this loop in case we want to change the quadrature at some point from the midpoint formula to doing a volume integral, which is something we talked about at AGU. I would therefore tend to leave it in this more general form but we could also take it out now and add it back in if we decide to test out other quadratures later. Both is fine with me, let me know what you prefer.